src/Contrib/Grpc/composer.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "open-telemetry/exporter-otlp-grpc",
+  "name": "open-telemetry/exporter-grpc",
```

```suggestion
  "name": "open-telemetry/transport-grpc",
```

Should remove `ext-protobuf`, `open-telemetry/exporter-otlp-common` and `open-telemetry/gen-otlp-protobuf` dependency.